src/katgpucbf/fgpu/recv.py

```diff
@@ -230,13 +231,17 @@ def make_sensors(sensor_timeout: float) -> aiokatcp.SensorSet:
                 "The timestamp (in samples) of the last chunk of data received from the digitiser",
                 default=-1,
                 initial_status=aiokatcp.Sensor.Status.ERROR,
+                auto_strategy=aiokatcp.SensorSampler.Strategy.EVENT_RATE,
+                auto_strategy_parameters=(SENSOR_UPDATE_TIME, math.inf),
```

Comment:
My main question is around what does the `math.inf` get interpreted as? Variable name aside, tracing the usage down to the [SensorSampler._send_update](https://github.com/ska-sa/aiokatcp/blob/master/src/aiokatcp/sensor.py#L368-L382)
* Specifically, `max(self.loop.time(), sched_time + self.longest)`

Reply:
It means that if the sensor value doesn't change, it won't spontaneously send an update to the client.